src/styles/product-card.scss

```diff
@@ -0,0 +1,73 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    color: $third-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__description {
+    color: $third-accent;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    margin-top: 16px;
+    display: flex;
+    padding: 12px 69px;
+    background-color: $second-accent;
+    cursor: pointer;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    font-size: 14px;
+    color: #fff;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      color: $second-accent;
+      background-color: #fff;
+      border-color: $second-accent;
```

Comment:
What about transition for this hover styles changes?

Reply:
what about transition? there is no word about transition in the task